lib/dpdk/CMakeLists.txt

```diff
@@ -1,23 +1,15 @@
 project(kotekan_libs)
 
-set(KOTEKAN_DPDK_SOURCES
-        dpdkCore.cpp
-        zeroSamples.cpp
-        invalidateVDIFframes.cpp
-        )
-
-add_library(kotekan_dpdk ${KOTEKAN_DPDK_SOURCES})
+add_library(kotekan_dpdk dpdkCore.cpp zeroSamples.cpp invalidateVDIFframes.cpp)
 
 # Link in DPDK
 find_library(dpdk REQUIRED)
 message("DPDK include dir: ${DPDK_INCLUDE_DIR}")
-include_directories(SYSTEM ${DPDK_INCLUDE_DIR})
-target_link_libraries(kotekan_dpdk ${DPDK_LIBRARIES})
-
-target_link_libraries(kotekan_dpdk libexternal)
+target_include_directories(kotekan_dpdk SYSTEM PRIVATE ${DPDK_INCLUDE_DIR})
+target_link_libraries(kotekan_dpdk PRIVATE ${DPDK_LIBRARIES} ${NUMA_LIBRARY} libexternal
+                                           kotekan_libs)
+target_include_directories(kotekan_dpdk PUBLIC .)
 
 add_definitions(-DWITH_DPDK)
 
-if(NOT ${NUMA_FOUND})
-    message( FATAL_ERROR "ERROR: libnuma is required for DPDK.  Please install libnuma")
-endif()
+find_package(NUMA REQUIRED)
```

Comment:
I think it works because cmake is odd, but maybe move this above line 9 since it's using a variable defined by this call.